lib/manager/dataset_manager.js

```diff
@@ -12,6 +12,9 @@ const { Connections, Connections2JSONLD } = require('gtfs2lc');
 const jsonldstream = require('jsonld-stream');
 const pageWriterStream = require('./pageWriterStream.js');
 const { GtfsIndex, Gtfsrt2LC } = require('gtfsrt2lc');
+const patriciaTree = require('rdf-patricia-tree');
```

Comment:
This dependency is not used here.

Reply:
This was meant for the vehicle tree, but vehicles are a `nice-to-have` in my thesis at the moment. I might add it later if my MVP experiments are complete. I will remove it for now and add the tree in another PR later.